uber/templates/base.html

```diff
@@ -130,12 +130,64 @@
         var parseQueryString = function(queryString) {
             var params = {}, temp, i, len, queries = queryString.split('&');
             for (i = 0, len = queries.length; i < len; i++) {
+              if(queries[i]) {
                 temp = queries[i].split('=');
                 params[temp[0]] = temp[1];
+              }
             }
             return params;
         };
 
+        var replaceQueryParam = function(queryParam, value) {
+            var queryString = window.location.search.substring(1);
```

Comment:
By the way, I think we might be able to do this in a more straightforward way if we render out cherrypy variables instead of relying on JS introspection of the DOM. I _think_ cherrypy just has a way to check if any params exist, so we could theoretically saying `if ({{ cherrypy.params }}) {}`.

Of course this is just moved code, so not changing the implementation in this PR was a good call.